tests/unit/test_adapter.py

```diff
@@ -1027,6 +1027,38 @@ def test_get_columns_in_relation_not_found_table(self):
         )
         assert columns == []
 
+    @mock_athena
+    @mock_glue
+    def test_delete_from_glue_catalog(self):
+        self.mock_aws_service.create_data_catalog()
+        self.mock_aws_service.create_database()
+        self.mock_aws_service.create_table("tbl_name")
+        self.adapter.acquire_connection("dummy")
+        relation = self.adapter.Relation.create(
+            database=DATA_CATALOG_NAME,
+            schema=DATABASE_NAME,
+            identifier="tbl_name",
+        )
+        self.adapter.delete_from_glue_catalog(relation)
+        glue = boto3.client("glue", region_name=AWS_REGION)
+        tables_list = glue.get_tables(DatabaseName=DATABASE_NAME).get("TableList")
+        assert tables_list == []
+
+    @mock_athena
+    @mock_glue
+    def test_delete_from_glue_catalog_not_found_table(self):
+        self.mock_aws_service.create_data_catalog()
+        self.mock_aws_service.create_database()
+        self.mock_aws_service.create_table("tbl_name")
+        self.adapter.acquire_connection("dummy")
+        relation = self.adapter.Relation.create(
+            database=DATA_CATALOG_NAME,
+            schema=DATABASE_NAME,
+            identifier="tbl_does_not_exist",
+        )
+        delete_table = self.adapter.delete_from_glue_catalog(relation)
+        assert delete_table is None
```

Comment:
Can you also assert the `caplog` has the correct log please ? Otherwise it does fully test the code. Also there is no case in the tests of an other error, which can be interesting